plugins/janus_videoroom.c

```diff
@@ -3349,6 +3362,28 @@ static void *janus_videoroom_handler(void *data) {
 						}
 					}
 				}
+				if(record_pause) {
```

Comment:
There is no check here on whether `arc`, `vrc` or `drc` exist. Trying to pause/resume a non existing recording should return an error, while in this case you're always succeeding no matter what.